src/convertToObject.js

```diff
@@ -7,6 +7,24 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const stylesLines = sourceString.split(';').map((item) => item.trim());
```

Comment:
Consider aligning chained method calls for better readability. Each method call should start on a new line indented to the same level.

Reply:
> Consider aligning chained method calls for better readability. Each method call should start on a new line indented to the same level.

Good point, and the next method you can to this chain as well

Reply:
@BudnikOleksii , can't do this with prettier.